src/components/HeroModal.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Card from './Card';
+import './Modal.css';
+
+function Modal({
+  heroList, heroFighterId, setHeroFighterId, show, setShow,
+}) {
+  return (
+    <div className={show ? 'heroSide' : 'hide'}>
+      <button type="button" onClick={() => setShow(false)}>X</button>
+      {heroList.map((hero) => (
+        <Card
+          key={hero.id}
+          avatar={hero}
+          fighterId={heroFighterId}
+          setFighterId={setHeroFighterId}
+        />
+      ))}
```

Comment:
use children to fill the modal, and reuse one Modal component for both Hero and Villain. something like that:

https://github.com/WildCodeSchool/reims-js-2103-project2-wild-code-tails/blob/main/src/component/CocktailList/Cocktail.jsx#L17
https://github.com/WildCodeSchool/reims-js-2103-project2-wild-code-tails/blob/main/src/component/CocktailList/Modal.jsx

Reply:
Modal changes done